setup-elixir/action.yml

```
@@ -1,43 +1,60 @@
 name: Setup Elixir
 description: Setup a specific version of Erlang/OTP and Elixir
+
 inputs:
   fetch-depth:
     description: Number of commits to fetch
     default: 1
   submodules:
     description: Whether to checkout submodules
     default: false
+  use-tool-versions:
+    description: Whether to use .tool-versions file for version management
+    default: false
   elixir-version:
     description: Version range or exact version of Elixir
-    required: true
+    required: false
   otp-version:
     description: Version range or exact version of Erlang/OTP
-    required: true
+    required: false
   cache-path:
     description: Paths that should be cached
     default: |
       _build
       deps
+
 outputs:
   cache-hit:
     description: A boolean value to indicate an exact cache was found
+
 runs:
   using: composite
   steps:
     - uses: actions/checkout@v4
       with:
         fetch-depth: ${{ inputs.fetch-depth }}
         submodules: ${{ inputs.submodules }}
-    - uses: erlef/setup-beam@v1
+
+    - name: Setup Erlang/OTP and Elixir using .tool-versions
+      if: ${{ inputs.use-tool-versions == 'true' }}
+      uses: erlef/setup-beam@v1
+      with:
+        version-file: '.tool-versions'
```

https://github.com/erlef/setup-beam?tab=readme-ov-file#version-file
测试下，把成功的 run 贴一下
```suggestion
        version-file: '.tool-versions'
        version-type: strict
```

---

https://github.com/Byzanteam/jet-storage-plugin/actions/runs/10980131585